docs/sources/clients/promtail/configuration.md

```diff
@@ -157,10 +157,9 @@ The `server` block configures Promtail's behavior as an HTTP server:
 [health_check_target: <bool> | default = true]
 ```
 
-## clients
+## client_config
```

Comment:
Are we just changing the header to match the `<client_config>` referenced above? If so should we do the same for `server` -> `server_config`, and `positions` -> `position_config`?

Reply:
Omg you right :facepalm: I got so used to the old syntax that I forgot to look at the other sections for comparison. Thank you!